Examples/minimum_user_for_s2e/src/src_user/Drivers/Com/gs.c

```diff
@@ -223,15 +220,43 @@ static DS_ERR_CODE GS_analyze_rec_data_(DS_StreamConfig* p_stream_config, void*
     driver_index = GS_PORT_TYPE_UART;
   }
 
-  gs_driver->info[driver_index].tc_frame_validate_status = GS_validate_tc_frame(tc_frame);
-  if (gs_driver->info[driver_index].tc_frame_validate_status != GS_VALIDATE_ERR_OK)
+  gs_driver->info[driver_index].tctf.tctf_validate_status = GS_validate_tctf(tctf);
+  if (gs_driver->info[driver_index].tctf.tctf_validate_status == GS_VALIDATE_ERR_OK)
   {
-    return DS_ERR_CODE_ERR;
+    rec = DS_ERR_CODE_OK;
   }
+  else
+  {
+    rec = DS_ERR_CODE_ERR;
+  }
+
+  gs_driver->info[driver_index].tctf.last_rec_tctf_type = TCTF_get_type(tctf);
+  switch (gs_driver->info[driver_index].tctf.last_rec_tctf_type)
+  {
+  case TCTF_TYPE_AD:
+    gs_driver->info[driver_index].tctf.ad_rec_status = rec;
+    break;
+
+  case TCTF_TYPE_BC:
+    gs_driver->info[driver_index].tctf.bc_rec_status = rec;
+    break;
+
+  case TCTF_TYPE_BD:
+    gs_driver->info[driver_index].tctf.bd_rec_status = rec;
+    break;
+
+  default:
+    // not reached
+    break;
+  }
+
+  if (rec == DS_ERR_CODE_ERR) return rec;
```

Comment:
[IMO]

エラーコード，増えないとも言えないので

```suggestion
  if (rec != DS_ERR_CODE_OK) return rec;
```